server/templates/obs.html

```diff
@@ -4,26 +4,35 @@
 {% block content %}
     <h3>Observation  {{ obs.obs_id }}</h3>
 
-    <p>
-        Thumbnail:<br/>
+    <p style="float: right">
```

Comment:
Use ```<div>``` or ```<span>```. ```<p>``` is for text. When we use Bootstap it may cause  not obvious problems.

Reply:
Using `div` now.